templates/public/views/bill.html

```diff
@@ -5,6 +5,9 @@
 {% block title %}{{ bill.identifier }} - {{ state|state_name }} {{ bill.from_organization.name }} ({{bill.legislative_session.identifier }}) - Open States{% endblock %}
 {% block og_title %}{{ bill.identifier }} - {{ state|state_name }} {{ bill.from_organization.name }} ({{bill.legislative_session.identifier }}) - Open States{% endblock %}
 {% block description %}Details on {{ state|state_name }} {{ bill.identifier}} ({{ bill.legislative_session }}) - {{ bill.title|titlecase_caps }}{% endblock %}
+{% block link %}
+    <link rel="canonical" href="https://pluralpolicy.com/app/legislative-tracking/bill/details/state-{{ state }}-{{bill.legislative_session.identifier|dash_check }}-{{ bill.identifier|cut:" "|lower }}" />
```

Comment:
Noticing the `state` part of `bill/details/states-{{`, which needs to be `federal` for USA bills. Can you do something 
 with the [yesno filter](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#yesno) or [if/else](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#if) to conditionally have it say state or federal in that string?